pulp_smash/resources/platform/api_v2/api.py

```diff
@@ -0,0 +1,183 @@
+# coding=utf-8
+"""Api package contains classes for easier resource managament in pulp-smash.
+"""
+
+from __future__ import unicode_literals
+
+import requests
+import time
+from requests.exceptions import HTTPError
+from pulp_smash.config import get_config
+
+# List of all paths here
+CREATE_REPOSITORY_PATH = "/pulp/api/v2/repositories/"
```

Comment:
This path is also used for searching. How about making a single constant like so:

```python
REPOSITORY_PATH = '/pulp/api/v2/repositories/'
```

This path can then be re-used throughout. If you'd really like to have separate constants for repositories in general and specific repositories, how about this?

```python
REPOSITORY_PATH = '/pulp/api/v2/repositories/'
REPOSITORY_ID_PATH = REPOSITORY_PATH + '{}/'
```